aip/general/0132.md

```diff
@@ -103,7 +103,8 @@ message ListBooksRequest {
 - The `page_token` field **must** be included on all list request messages.
 - The request message **may** include fields for common design patterns
   relevant to list methods, such as `string filter` and `string order_by`.
-- The request message **must not** contain any other required fields, and
+- The request message **must not** contain any other required fields (except 
+  for `parent`, `page_size` and `page_token` fields explained above), and
```

Comment:
Why make "(except...above)" a parenthetical? It seems important/non-optional.

Reply:
Good point. Done.